resources/cf-url-rewriter/cf-url-rewriter.ts

```diff
@@ -19,9 +19,16 @@ export async function handler(event: CloudFrontRequestEvent, context: Context, c
 
   if (request.uri.includes("/latest/")) {
 
-    const indexUri = request.uri.replace(/\/latest\/.*/, '/index.json');
+    const urisplit = request.uri.split("/latest");
+    const index = urisplit[0].lastIndexOf("/");
+    const versionNumber = urisplit[0].substring(index + 1).replace(/\./g, '');
```

Comment:
nonblocking: all these variables are used for `getIndexFilePath` and not in the main function. Better to move them into the sub function getIndexFilePath